evm_arithmetization/src/cpu/kernel/asm/core/process_txn.asm

```diff
@@ -324,46 +329,83 @@ process_message_txn_fail:
     %transfer_eth %jumpi(panic)
     %jump(process_message_txn_after_call_contd)
 
-%macro pay_coinbase_and_refund_sender
-    // stack: leftover_gas
+global pay_coinbase_and_refund_sender:
+    // stack: leftover_gas, retdest
```

Comment:
I'm not sure I understand this. The only syntactic different is
```asm
    #[cfg(feature = cdk_erigon)]
    {
        %deduct_extra_burn_fees
    }
```
at the end on `leftover_has'` right? Why not just calling this within the macro?

Reply:
I tried calling it within the macro, but I got the error that it should have been parsed before (I don't think we can have features within a macro with the current implementation)

Reply:
Hmm this is a parsing issue then, possibly due to ordering, that should be dealt with. We would want to have conditional attributes within macros too